src/pages/DataPage.svelte

```diff
@@ -72,11 +72,11 @@
   }
 
   let trainingButtonPrimary = false;
-  gestures.subscribe(gestures => {
-    if (gestures.filter(g => g.recordings.length >= 3).length >= 2) {
+  $: {
+    if ($gestures.filter(g => g.recordings.length >= 3).length >= 2) {
       trainingButtonPrimary = true;
     }
-  });
```

Comment:
Potential alternative is to merge this with logic from the training button. Then use a `<StandardButton />` in place of `<TrainingButton />` to avoid duplication in DataPage. `<TrainingButton />` is used elsewhere.
 ```
  // Workaround: hasSufficientData uses gestures but isn't reactive
  $: sufficientData = $gestures && hasSufficientData();
  $: trainingButtonDisabled = !sufficientData || $state.isTraining;
  let trainingButtonPrimary = false;
  $: {
    trainingButtonPrimary = trainingButtonPrimary || sufficientData;
  }
```